js/controllers/todos_controller.js

```diff
@@ -1,4 +1,125 @@
-App.TodosController = Ember.ObjectController.extend({
+
+Abs.TasksController = Ember.ObjectController.extend({
     actions: {
+        editTasks: function () {
+            this.set('editMode', true);
+        },
+        changeTask: function () {
+            var me = this;
+            Ember.run.later((function () {
```

Comment:
I wrote it to handle this error :"Uncaught Error: Attempted to handle event `willCommit`"when I change textbox to edit task and enter it had this error, but when I leave focus didn't have.
 


     On Monday, September 28, 2015 5:16 PM, Ruan Naude <notifications@github.com> wrote:
   

 In js/controllers/todos_controller.js:>      actions: {
> +        editTasks: function () {
> +            this.set('editMode', true);
> +        },
> +        changeTask: function () {
> +            var me = this;
> +            Ember.run.later((function () {
Why is this on an Ember later function?—
Reply to this email directly or view it on GitHub.